### DISCUSSION TITLE
Request to rename the top level: brand tokens.

### DISCUSSION THREAD

**Discussion by AlineNap** [nl-design-system/backlog#203] [Design token suggestions]
Tokens zijn onderverdeeld in 3 lagen:

- Top level: brand
- Mid: common
- Lowest: component

Bij LUX, Design system van Logius hebben we 1 common laag die verschillende brands dient, oa DigiD, MijnOverheid en Logius. Daarvoor hebben we op common niveau een color token die per brand een andere alias krijgt. Deze color token heet momenteel lux.color.brand.default*. Dit levert verwarring op doordat de toplaag ook brand heet. Als oplossing kan de token worden hernoemd naar lux.color.product.default, maar product dekt niet altijd de lading en LUX zal niet het enige systeem zijn die hier tegenaan loopt. Ook open gemeentes zal meerdere brands dienen. 

Is het een idee om de top level te hernoemen? Bijvoorbeeld core, primatives of global tokens? 
Hier een link naar een FigJam van Samantha Gordashko die verschillende benamingen in kaart heeft gebracht: https://www.figma.com/community/file/1319771374095138011

_*Naamgeving zit niet in stramien van NL design system, dat is een ander topic._

**Comment by Robbert** [comment]
@AlineNap `brand` heeft als voornaamste voordeel dat met zo'n prefix alle fundamentele kleuren bij elkaar staan. De "vendor prefix", het allereerste deel van de design token naam, kun je gebruiken om kleuren van verschillende merken apart te zetten.

Je zou zo'n onderverdeling kunnen doen:

Brand tokens:
- Logius: `logius.brand.color.hemelblauw`
- DigiD: `digid.brand.color.gezelligoranje`

Die kun je ook weer op hun eigen manier toewijzen aan common tokens en component tokens.
